docs/data_quality.md

```diff
@@ -0,0 +1,114 @@
+# Properties of a High Quality OSV Record
+
+## Version
+
+1.0.0 (SEMVER)
+
+## Purpose
+
+Describe the “good enough” OSV record that will be imported by OSV.dev
+
+### Out of scope
+
+This does not discuss the problem of record bit rot over time, after initial successful import. The problem of continuous revalidation and treatment of records that have been successfully imported will be dealt with separately in .
+
+Deferred to a future iteration: validating the existence of vulnerable functions in the `ecosystem_specific` field, if supplied.
+
+## Audience
+
+1. Current and aspiring OSV record producers
+2. Downstream OSV.dev record consumers
+
+## Rationale
+
+OSV.dev seeks to be an comprehensive, accurate and timely database of known vulnerabilities (that is highly automation friendly). In order to meet this accuracy goal, a quality bar needs to be both defined and sustainably enforced.
+
+## Properties of a High Quality OSV Record
+
+### Valid
```

Comment:
nit, but It's probably also worth adding that the issue in the record itself is valid.

Reply:
I'm sorry, I'm not understanding what you're saying here. Could you perhaps make a suggested change to the content to clarify?

Reply:
Sure thing. Added one here
https://github.com/google/osv.dev/pull/2193/files#r1621522388

The idea is to state explicitly that lies/untruths/etc... are not high quality.

Reply:
Thanks!